liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ApplicationXmlDocument.java

```diff
@@ -37,46 +37,55 @@ public ApplicationXmlDocument() {
             e.printStackTrace();
         }
     }
-    
-    public void createApplicationElement(String appFileName, String artifactId) {
+
+    public void createApplicationElement(String appFileName, String artifactId, boolean thin) {
```

Comment:
Instead of changing the signature of this public method, I think you should add a new **createApplicationElement** method that takes the `boolean thin` parameter, and have this existing method call it and pass the default (which I presume would be **false**).

Reply:
Done.